index.bs

```diff
@@ -549,8 +549,16 @@ interface WebTransport {
   Promise&lt;WebTransportSendStream&gt; createUnidirectionalStream();
   /* a ReadableStream of WebTransportReceiveStream objects */
   readonly attribute ReadableStream incomingUnidirectionalStreams;
+
+  /* of type CongestionControlFeedbackEvent */
+  attribute EventHandler oncongestioncontrolfeedback;
 };
 
+interface CongestionControlFeedbackEvent: Event {
+  readonly attribute unsigned long? targetBps;
```

Comment:
The design principle [§ 7.7. Use plain Events for state](https://w3ctag.github.io/design-principles/#state-and-subclassing) would suggest:
```js
wt.oncongestioncontrolfeedback = () => foo(wt.targetBps);
```

...which I suppose begs the question of why not
```js
wt.oncongestioncontrolfeedback = async () => foo((await wt.getStats()).targetBps);
```
...which in turn leaves me wondering why we need an event if it's firing on a fixed cadence...
```js
setInterval(async () => foo((await wt.getStats()).targetBps), 100);
```
Or is the idea it only fires when congestion changes? That's really the only way events "avoids pulling".

Reply:
The argument for having `targetBps` be an event rather than another statistic is that `targetBps` might be needed more frequently than the rest of the statistics. If so, then you'd avoid polling `getStats()` more frequently than necessary. 

[Salsify ](https://snr.stanford.edu/salsify/) argues for the ability to adjust bandwidth usage on frequent basis (possibly per frame), based on available bandwidth. The adjustment could be made by changing per-frame QP (in progress in WebCodecs) or dropping/adding an extension layer.  Salsify makes an argument that frequent adjustment yields better video quality than adjusting the average target bitrate every keyframe.

Reply:
The idea is that the event would only fire when the value changes, so calling .getStats() doesn't make sense, nor does a fixed cadence.

Reply:
Which somewhat contradicts what I wrote in the PR :).  I don't remember why I put in an interval.  I think I'll just change it to "fire when it changes".

Reply:
I made it a max interval so if it doesn't change, it doesn't fire, and if it changes a lot, it will be throttled.  Hopefully that's the best of both worlds (avoid polling and avoid too often of an event).

Reply:
This if fixed (an event with no payload plus a property).